code/game/objects/items/emags.dm

```diff
@@ -14,8 +14,16 @@
 	item_flags = NO_MAT_REDEMPTION | NOBLUDGEON
 	slot_flags = ITEM_SLOT_ID
 	worn_icon_state = "emag"
-	var/prox_check = TRUE //If the emag requires you to be in range
-	var/type_blacklist //List of types that require a specialized emag
+	/// If the emag requires you to be in range
+	var/prox_check = TRUE
+	/// List of types that require a specialized emag
+	var/type_blacklist
+	/// If the emag can be used by crewmembers
+	var/useable_about_crew = FALSE
```

Comment:
The spelling mistake aside, the vars not terribly accurate. You can use it as crew, but it is dangerous to do so.

Probably better to call it ``has_theft_countermeasure`` or something.